### PR TITLE
feat(snackbar): new component

### PR DESCRIPTION
**Pull request by DeadEnglish**
## Screenshot / video

https://user-images.githubusercontent.com/5758372/223498603-3bd08387-ef33-47ed-97f2-efc59daca4ab.mov

## What does this do?

New Snackbar component based on the [Snackbar S'mores design](https://www.figma.com/file/FqSaizGOyOzXFZNmZ4X0LGMn/%F0%9F%8D%AC-S'mores?node-id=7616%3A25308&t=EzatA7hjIG8N5CNX-1)

### CODE REVIEW

File: src/Snackbar/SnackbarContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+export const SnackbarContainer: FC<{ children?: ReactNode }> = ({
+  children,
+}) => {
+  let snackbarId = 0
+  const [snackbars, setSnackbars] = useState<Snackbar[]>([])
+
+  // create snackbar unique ID
+  const newSnackId = useCallback(() => {
+    const newId = `MM_SNACKBAR_${snackbarId}`
```
Comment (fleonard) [inline src/Snackbar/SnackbarContainer.tsx:29]: IIRC we have a unique ids generator in utils

**Review by fleonard — COMMENTED** [review]
Looks pretty god to me, just left a minor comment. I suppose we can go as it is for now and then build on top of it when design has figured out what they wanna do about statusses

File: src/Snackbar/SnackbarContainer.tsx (hunk 1 above)
Comment (DeadEnglish) [reply src/Snackbar/SnackbarContainer.tsx:29]: @fleonard I did try and re-use it but due to it being a custom hook, you can't use it within the `addSnackbar` function 😢

Comment (fleonard) [reply src/Snackbar/SnackbarContainer.tsx:29]: oh damn - fair enough!

**Review by fleonard — APPROVED** [review]

**Comment by DeadEnglish** [comment]
[RFC](https://www.notion.so/getmarshmallow/Snackbar-4da6fb3ea8ea4f89a84e185cf8af012a?pvs=4)

**Pull request merged**